server/models/request.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "user",
+    },
+
+    sitter: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "sitter",
```

Comment:
```suggestion
      ref: "user",
```
We don't have a separate model for sitters :)